NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# a user can create a tag

#### Changes Made
1. Added file `tagForm.js` to `Tag` directory.
2. Added to `ApplicationViews.js` file to include proper  routing for new tag.
3. Refactored `TagList.js` file to display `a list of tags` instead of using `tag.js`.
4. Refactored `TagController.cs`, `Tag.cs` and `TagRepository.cs` to be more uniform
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sr-create-tag
    ```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > Inside your `Client directory`
            `Npm Start`
    > Run the `web api`
    > Example: When user does `submits a new tag`, then `the user should be directed to a list of tags`.
6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
